specification/ai/Azure.AI.Projects/tspconfig.yaml

```diff
@@ -18,6 +20,23 @@ options:
     flavor: azure
     generate-test: true
     generate-sample: false
+  "@azure-tools/typespec-csharp":
+    emitter-output-dir: "{csharp-service-dir}/{namespace}"
+    package-mode: "dataplane"
+    namespace: "Azure.AI.Projects"
+    package-name: "{package-dir}"
+    api-version: "2025-05-15-preview"
+    model-namespace: false
+    flavor: azure
+    generate-test: false
+    generate-sample: false
+    generate-sample-project: false
+    clear-output-folder: true
+  "@azure-typespec/http-client-csharp":
+    emitter-output-dir: "{csharp-service-dir}/{namespace}"
+    namespace: Azure.AI.Projects
+    model-namespace: false
+    api-version: "2025-05-15-preview"
   "@typespec/http-client-csharp":
```

Comment:
Should this be removed?

Reply:
No, this library uses the unbranded emitter so this entry is correct. The other two should be removed.